desktop/cmp/dataview/DataViewModel.js

```diff
@@ -40,11 +44,32 @@ export class DataViewModel {
         emptyText,
         contextMenuFn = null
     }) {
+
+        selModel = this.parseSelModel(selModel, store);
```

Comment:
This was pre-existing and applies to GridModel as well.  Given that these ctors can take a StoreSelectionModel instance or a config/string to create, we might/might not want to clean those models up when disposing of the parent model. 

I suspect most of the time we're creating them. Is this something we should be sure to handle?

I also realize I don't have a good grasp on the impact of _not_ destroying something like a StoreSelectionModel....